integration-tests/wallet-multisig-scenario.spec.ts

```diff
@@ -5,12 +5,12 @@ import { genericMultisig } from "./data/multisig";
 CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
   const Tezos = lib;
 
-  describe(`Generic Multisig: ${rpc}`, () => {
+  describe(`Test originating multisig contract through wallet api: ${rpc}`, () => {
```

Comment:
Not only an origination, but also interacting with the contract to do a transfer to an implicit account

Reply:
changed to
"Test multisig contract through wallet api for origination, contract interaction, and tranfer to an implicit account with: ${rpc}"